### PR TITLE
[RFR] Fix webpack configuration and updated admin

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: app/admin/js/header.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="navbar-header">
+    <button type="button" class="navbar-toggle" ng-click="isCollapsed = !isCollapsed">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+    </button>
+    <a class="navbar-brand" href="#" ng-click="appController.displayHome()">Autocast backend</a>
```
Comment (Kmaschta) [inline app/admin/js/header.html]: Autocast

**Comment by Kmaschta** [comment]
:+1:

File: app/admin/js/main.js [hunk 2]
```diff
@@ -2,15 +2,26 @@
...
+window.logout = logout;
+
+if (!window.sessionStorage.getItem('token')) redirectToLogin();
+
 const myApp = angular.module('myApp', ['ng-admin']);
 
 myApp.config(['NgAdminConfigurationProvider', (nga) => {
     const admin = nga
-        .application('TeamPlay')
+        .application('New App')
```
Comment (alexisjanvier) [inline app/admin/js/main.js]: Could `App name` came from the config ?

Comment (alexisjanvier) [inline app/admin/js/main.js]: (i'm not sure that's important ...)

**Comment by alexisjanvier** [comment]
:+1:

File: app/admin/js/main.js (hunk 2 above)
Comment (Kmaschta) [inline app/admin/js/main.js]: Good idea :+1: 
I'll do this in my next PR

**Comment by Kmaschta** [comment]
:+1:

File: app/admin/js/header.html (hunk 1 above)
Comment (alexisjanvier) [inline app/admin/js/header.html]: FMI, what do you mean with `Autocast`

**Pull request merged**